src/MagnumPlugins/UfbxImporter/UfbxImporter.h

```diff
@@ -53,6 +53,241 @@
+    importer-specific @cpp "Visibility" @ce (of type @ref SceneFieldType::UnsignedByte
+    representing a boolean value).
+    These five fields share the same object mapping with
+    @ref SceneFieldFlag::ImplicitMapping set.
+-   If the scene references meshes, a @ref SceneField::Mesh (of type
+    @ref SceneFieldType::UnsignedInt) and a @ref SceneField::MeshMaterial (of
+    type @ref SceneFieldType::Int) is present, both with
+    @ref SceneFieldFlag::OrderedMapping set. Missing material IDs are @cpp -1 @ce.
+    If a mesh contains multiple materials it is split into parts and the node
+    contains each part as a separte mesh/material entry.
+    The same mesh can appear instanced multiple times under many
+    nodes with different materials. See @ref Trade-UfbxImporter-instancing for further details.
+-   If the scene references cameras, a @ref SceneField::Camera (of type
+    @ref SceneFieldType::UnsignedInt) is present, with
+    @ref SceneFieldFlag::OrderedMapping set. A single camera can be referenced
+    by multiple nodes. See @ref Trade-UfbxImporter-instancing for further
+    details.
+-   If the scene references lights, a @ref SceneField::Light (of type
+    @ref SceneFieldType::UnsignedInt) is present, with
+    @ref SceneFieldFlag::OrderedMapping set. A single light can be referenced
+    by multiple nodes. See @ref Trade-UfbxImporter-instancing for further
+    details.
+
+@subsection Trade-UfbxImporter-behavior-materials Material import
+
+-   Supports both legacy FBX Phong material model and more modern PBR
+    materials, in some cases both are defined as PBR materials may have
+    a legacy Phong material filled as a fallback.
+-   The legacy FBX material model and most PBR material models have factors
+    for various attributes, by default these are premultiplied into the value
+    but you can retain them using the @cb{.ini} preserveMaterialFactors @ce @ref Trade-UfbxImporter-configuration "configuration option"
+-   Two-sided property and alpha mode is not imported.
+-   ufbx tries to normalize the various vendor-specific PBR material modes into
+    a single set of attributes that are imported, see @ref Trade-UfbxImporter-pbr-attributes
+    for an exhaustive listing.
+-   @ref MaterialAttribute::DiffuseTextureMatrix and similar matrix attributes
+    for other textures are imported.
+-   @ref MaterialAttribute::DiffuseTextureCoordinates and similar UV layer
+    attributes are not supported.
+
+@subsection Trade-UfbxImporter-behavior-lights Light import
+
+-   @ref LightData::Type::Directional and @ref LightData::Type::Ambient expect
+    the attenuation to be constant, but FBX is not required to follow that.
+    In that case the attenuation value from the file is ignored.
+-   Area and volume lights are not supported
+
+@subsection Trade-UfbxImporter-behavior-meshes Mesh import
+
+-   Only triangle meshes are loaded (quad and poly meshes are triangularized)
```

Comment:
There's two ways FBX can contain lines/points. First one is just having degenerate polygons in a mesh, Blender has an option for preserving these. Would be possible to support those but we'll need to split the meshes into further chunks then. ufbx already splits it across materials to `ufbx_mesh_material::face_indices[]` as that's needed so often, but for lines/points we'd need to scan each mesh chunk and split it into three according to the number number of indices per face. ~~It seems like it would be a lot of bookkeeping for that~~ Actually scratch that, I just figured out an easy and elegant way of supporting this, wouldn't even need any modification from ufbx side but I'm already counting the number of triangles in a linear pass so might as well count the number of points/lines while I'm at it.

The second way is `ufbx_line_curve` which doesn't support materials (to my understanding, ie. haven't been able to make a file with a real program. Those would be pretty easy to include but again pretty rare in practice, I've only managed to make them with extensively messing around 3ds Max. There's more common `ufbx_nurbs_curve` that ufbx can tessellate into `ufbx_line_curve`s but that might get a bit out of scope :D

Reply:
I see, so it's not really a builtin support, thanks for the explanation. Can you clarify the docs to say that? Because now it reads like "FBX supports lines/points but this plugin imports only triangles". Something like 

```suggestion
-   Polygon meshes are triangulated, degenerate polygons are kept as-is without
    being converted to lines or points. FBX doesn't have native support for 
    line/point meshes.
```

Reply:
Always retain them, yes :) Awesome.